alacritty.yml

```diff
@@ -497,45 +501,79 @@
 #   - ScrollToTop
 #   - ScrollToBottom
 #   - ClearHistory
+#       Remove the terminal's scrollback history.
 #   - Hide
+#       Hide the Alacritty window.
 #   - Minimize
+#       Minimize the Alacritty window.
 #   - Quit
+#       Quit Alacritty
 #   - ToggleFullscreen
 #   - SpawnNewInstance
+#       Spawn a new instance of Alacritty that preserves the path. Only applies to Linux/BSD.
```

Comment:
It's not really true that this action only applies to Linux/BSD, but only on Linux/BSD do we currently try to open it in the shell's current working directory.

It's also not guaranteed to work, for several technical reasons.

Reply:
Personally I'd avoid mentioning the path at all here. It's unreliable across systems. Perhaps if, down the road, launching in the same working directory works more consistently, then we could mention it here.

Reply:
@nixpulvis Does it work reliably on Linux? For those of us who use Linux exclusively, that's interesting to us. It seems fair to document features that only work on one platform or environment.

In the interest of getting this merged, though I'll drop the comment since both of you spoke against it.

Reply:
TBH I haven't tested it in a while, I setup an X11 dependent script a while back to use as a launcher, and never really needed to replace it (though it would be nice to remove it).

FWIW, here's my script:

```sh
ACTIVE_WINDOW=$(xdotool getactivewindow)
ACTIVE_WM_CLASS=$(xprop -id $ACTIVE_WINDOW | grep WM_CLASS)

if [[ $ACTIVE_WM_CLASS == *"Alacritty"* ]]
then
    # Get PID. If _NET_WM_PID isn't set, bail.
    PID=$(xprop -id $ACTIVE_WINDOW | grep _NET_WM_PID | grep -oP "\d+")
    if [[ "$PID" == "" ]]
    then
        exit 1
    fi

    # Get first child of terminal
    CHILD_PID=$(pgrep -P $PID)
    if [[ "$PID" == "" ]]
    then
        exit 1
    fi

    # Get current directory of child. The first child should be the shell.
    pushd "/proc/${CHILD_PID}/cwd"
    SHELL_CWD=$(pwd -P)
    popd

    # Start alacritty with the working directory
    alacritty --working-directory $SHELL_CWD $@
else
    exit 1
fi
```

Reply:
Thanks for the tip. I'm marking this as resolved because I removed the statement about the path for now.